### ISSUE TITLE
List - examples should have unique aria labeling

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7287]
**Describe the enhancement or change**
The lists in examples should have unique aria labeling to follow the recommendation of doing so when multiple are on a page. For nested lists, the label could be the text of its parent item if it isn't verbose or examples could be updated to follow the "Inline" example, i.e. "X Item 1..." instead of using lorem as the list item content.

**Any other information?**

**Comment by thatblindgeye** [comment]
Similar to React counterpart, going to close this for now and ease back on list labeling after discussion with another a11y expert
